profiles/arch/arm/package.use.mask

```diff
@@ -509,3 +509,7 @@ dev-qt/qtscript -jit
 
 # missing keywords
 media-plugins/gst-plugins-meta aac dts dv lame libvisual modplug vcd wavpack
+
+# Anthony Ryan <anthonyryan1@gmail.com> (2024-10-13)
+# Test dependency dev-libs/stb not keyworded on this arch.
+media-libs/zxing-cpp test
```

Comment:
Why not dekeyword and request rekeywording?

Reply:
Because I was told by Gentoo developers in the past that it was better to mask a new use flag, rather than to drop a keyword for the whole package because historically packages don't always get keyworded again.